src/sentry/api/base.py

```diff
@@ -108,17 +108,16 @@ def create_audit_entry(self, request, **kwargs):
             **kwargs
         )
 
+        extra = {
+            'entry_id': entry.id,
+            'actor_label': entry.actor_label
+        }
         if entry.actor_id:
-            logger.bind(actor_id=entry.actor_id)
+            extra['actor_id'] = entry.actor_id
         if entry.actor_key_id:
-            logger.bind(actor_key_id=entry.actor_key_id)
+            extra['actor_key_id'] = entry.actor_key_id
 
-        logger.info(
-            name='sentry.audit.entry',
```

Comment:
Probably should add some context, but I changed the names of the loggers
for both this and 'sentry.web.frontend.base' to be their respective
loggers. That way we can actually tell the path of the entry created.

On Wed, Jul 13, 2016 at 4:15 PM Matt Robenolt <notifications@github.com>
wrote:

> In src/sentry/api/base.py
> <https://github.com/getsentry/sentry/pull/3714#discussion_r70725078>:
>
> >
> > -        logger.info(
> > -            name='sentry.audit.entry',
>
> Aren't we losing this now? I don't see how this would be retained.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/getsentry/sentry/pull/3714/files/4a372c3e83f8972bb8e3879bc66fe04a30f7ff24#r70725078>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABImRkiAWVBurts5dVRL7qAw_gJEr9xdks5qVXGegaJpZM4JL3Di>
> .
>